crates/sui-graphql-rpc/src/types/cursor.rs

```diff
@@ -298,8 +304,14 @@ impl<C: CursorType + Eq + Clone + Send + Sync + 'static> Page<C> {
         };
 
         Ok(self.paginate_results(
-            results.first().map(|f| f.cursor()),
-            results.last().map(|l| l.cursor()),
+            results.first().map(|f| match checkpoint_viewed_at {
+                Some(checkpoint_viewed_at) => f.consistent_cursor(checkpoint_viewed_at),
+                None => f.cursor(),
```

Comment:
I don't think this is right -- we should always have a `checkpoint_viewed_at` for `paginate_raw_query` -- it's either a parameter that's passed down, or `rhs` of the available range.

Reply:
Ah yes. it didn't occur to me that I was always passing Some(checkpoint_viewed_at), that I should instead change the signature of this parameter.